app/models/user.rb

```diff
@@ -1,8 +1,11 @@
 class User < ApplicationRecord
   has_secure_password
   has_many :orders
+
+  validates :name, presence: true
   validates :username, presence: true,
                        uniqueness: true
+  validates :password, presence: true
```

Comment:
I think that has_secure_password automatically includes a validation for the presence of a password (http://apidock.com/rails/v4.0.2/ActiveModel/SecurePassword/ClassMethods/has_secure_password), so this line isn't necessary.